packages/jaeger-ui/src/components/DeepDependencies/index.tsx

```diff
@@ -173,7 +173,7 @@ export class DeepDependencyGraphPageImpl extends React.PureComponent<TProps, TSt
     const { graphState } = this.props;
     const { visEncoding } = this.props.urlState;
 
-    if (graphState && graphState.state === fetchedState.DONE) {
+    if (graphState && graphState.state === fetchedState.DONE && 'model' in graphState) {
```

Comment:
this doesn't feel like the right solution to me. Is the reason for linting errors due to missing or incomplete type declaration? Based on the existing code it looks like the `model` field is always expected under these conditions, so why do we need to test for it at runtime, instead of enforce it via type checking?

Reply:
Yes,this is the reason for linting errors. Yeah,the is approach in not intuitive. Let me fix it.

Reply:
The problem is [TS2339: Property does not exist on type {}]

Reply:
why can't we add it to the type declaration?

Reply:
Yes, I've added it to the type declaration, now it's fine. Let me fix the PR.